controllers/constants/constants.go

```diff
@@ -19,4 +19,13 @@ const (
 	VCLUSTER_K8S_DISTRO         = "k8s"
 	K3S_DATASTORE_ENDPOINT      = "K3S_DATASTORE_ENDPOINT"
 	VCLUSTER_INGRESS_HOSTNAME   = "VCLUSTER_INGRESS_HOST"
+	DEFAULT_K3S_VERSION         = "rancher/k3s:v1.27.3-k3s1"
+	UCLUSTER_SYNC_PLUGIN_TAG    = "uffizzi/ucluster-sync-plugin:v0.2.4"
+	OCI_TYPE                    = "oci"
+	PREMIUM_RWO_STORAGE_CLASS   = "premium-rwo"
+	NO_SCHEDULE                 = "NoSchedule"
```

Comment:
This is available within the k8s.core.v1 module as [v1](https://pkg.go.dev/k8s.io/api/core/v1).[TaintEffectNoSchedule](https://pkg.go.dev/k8s.io/api/core/v1#TaintEffectNoSchedule)

but I don't know if that module is available to everywhere that needs it below

Reply:
it is available, I am using it now